src/components/global/Playground/README.md

```diff
@@ -46,15 +46,17 @@ import javascript from './javascript.md';
 />;
 ```
 
-The file location is respective to the Stackblitz demo project. If you are unsure of the exact path, refer to the `stackblitz.utils.ts` method for each respective target. The path specified in `sdk.openProject` will coincide with the value used in the `files` configuration.
+<!-- cspell:disable-next-line -->
```

Comment:
Why was this skip added? I can only think of the bits in `code`, but I assumed the second regex in the ignorelist was to handle things like that.

Reply:
It throws an error on `stackblitz` in `stackblitz.utils.ts`:

```
ionic-docs/src/components/global/Playground/README.md:50:116 - Unknown word (stackblitz)
```

I didn't want to add `stackblitz` as an ignored word (all lowercase) because currently `cspell` errors and requires it to be written as `StackBlitz` which prevents us from writing it `Stackblitz` or `stackblitz` in normal paragraphs.

Maybe there is a way to determine if it's part of a filename. There are some other issues with file names, most notably this one:

```
ionic-docs/docs/angular/pwa.md:37:45 - Unknown word (ngsw)
```

This error comes from here: `"source": "ngsw-worker.js",`

I could look into writing a regex that ignores words ending in `*.ts` or `*.js` maybe.

Reply:
Sorry I missed your second part. You are right the regex for code ticks should be catching it. I'll look into this more!

Reply:
I modified the regex to include file names and the `@` symbol which allowed me to remove a lot of flagged words as well that were in code ticks but not being ignored: https://github.com/ionic-team/ionic-docs/pull/3069/commits/a58bc90138330fc59dcbb9b4725e0738ca7ea5e3

Reply:
Expanded the regex to match anything inside of two code ticks, this allowed me to remove a lot of the ignored words in. I updated the docs to wrap some of the flagged words that could be in code formatting. Broken into several commits:

- Adding backticks around datetime formats: https://github.com/ionic-team/ionic-docs/pull/3069/commits/1e0f260709fcc16010372b45b03806410f55f73f
- Adding backticks around slide events (Swiper migration guide): https://github.com/ionic-team/ionic-docs/pull/3069/commits/bf865f266b0060499f7c4664e527f21559e85258
- Adding backticks around keyboard events: https://github.com/ionic-team/ionic-docs/pull/3069/commits/3216673bc16b870a1acf5b28d549a0671e3da2b7
- Adding backticks around `rgba`: https://github.com/ionic-team/ionic-docs/pull/3069/commits/c31477bdef1f73ef2aa3622c1bea253e15fe1a80
- Updating regex to include everything between backticks: https://github.com/ionic-team/ionic-docs/pull/3069/commits/91635140c7616b3bc608a32c0e096fbc98ac601b